metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorPartitionService.java

```
@@ -105,26 +104,16 @@ public HiveConnectorPartitionService(
     public List<PartitionInfo> getPartitions(
         final ConnectorRequestContext requestContext,
         final QualifiedName tableName,
-        final PartitionListRequest partitionsRequest
-    ) {
-        try {
-            final List<Partition> partitions = getPartitions(tableName,
-                partitionsRequest.getFilter(), partitionsRequest.getPartitionNames(),
-                partitionsRequest.getSort(), partitionsRequest.getPageable());
-            final Table table = metacatHiveClient.getTableByName(tableName.getDatabaseName(), tableName.getTableName());
-            final TableInfo tableInfo = hiveMetacatConverters.toTableInfo(tableName, table);
-            final List<PartitionInfo> partitionInfos = new ArrayList<>();
-            for (Partition partition : partitions) {
-                partitionInfos.add(hiveMetacatConverters.toPartitionInfo(tableInfo, partition));
-            }
-            return partitionInfos;
-        } catch (NoSuchObjectException exception) {
-            throw new TableNotFoundException(tableName, exception);
-        } catch (MetaException | InvalidObjectException e) {
-            throw new InvalidMetaException("Invalid metadata for " + tableName, e);
-        } catch (TException e) {
-            throw new ConnectorException(String.format("Failed get partitions for hive table %s", tableName), e);
```

What is the reason to ignore this exception?

---

These exceptions are throw from the metacatHiveClient.getTableByName() call, which won't be needed after we passed in the tableInfo.